src/frontend/README.md

```diff
@@ -26,29 +26,15 @@ In the project directory, you can run:
 VITE_API_SERVER=<BACKEND_URI>
 ```
 
-#### Option 1. Build and deploy to Google Cloud Run in one step
+#### Build and deploy to Google Cloud Run in one step
 
 ```
 gcloud run deploy image-processing-frontend --source . --region=us-central1 --cpu=2 --memory=8G --timeout=3600 --allow-unauthenticated
 ```
 
-This packages the frontend into an image using the Dockerfile and saves it in the Google Container Registry. It then deploys the image right away.
+This packages the frontend into an image using the Dockerfile and saves it in the Artifact Registry. It then deploys the image right away.
 
-#### Option 2. Build a container using Google Cloud Build
```

Comment:
I'm removing option 2. Uses can just use option 1 (`gcloud run deploy ... --source`) with the `--source` argument to build and deploy in a single step.